src/vapid-helper.js

```diff
@@ -6,6 +6,12 @@ const asn1 = require('asn1.js');
 const jws = require('jws');
 const url = require('url');
 
+/**
+ * DEFAULT_EXPIRATION is set to seconds in 12 hours
+ */
+const DEFAULT_EXPIRATION_SECONDS = 12 * 60 * 60;
+const MAX_EXPIRATION_SECONDS = 24 * 60 * 60;
```

Comment:
I think VAPID must have an expiration less that 24 hours, but link to spec would be good here.